src/explore-education-statistics-admin/src/pages/release/data/components/AncillaryFileForm.tsx

```diff
@@ -113,6 +114,7 @@ export default function AncillaryFileForm({
         summary: Yup.string().required('Enter a summary'),
         file: Yup.file()
           .minSize(0, 'Choose a file that is not empty')
+          .maxSize(2147483648, 'Choose a file that is under 2GB')
```

Comment:
I would have said a similar comment to the C# code and putting this in a named constant, but given that it's right next to the error message that tells you what's going wrong, I think we're good here :slightly_smiling_face: